### PR TITLE
SWITCH: Implement OSystem_Switch::getSystemLanguage()

### PR DESCRIPTION
**Pull request by usineur**
This commit adds language detection on Switch

### CODE REVIEW

**Comment by usineur** [comment]
@rsn8887 can you review this pls ?

**Comment by rsn8887** [comment]
Looks good to me.

**Comment by rsn8887** [comment]
Merged, thank you!

File: backends/platform/sdl/switch/switch.cpp [hunk 1]
```diff
@@ -164,3 +166,29 @@ Common::HardwareInputSet *OSystem_Switch::getHardwareInputSet() {
...
+
+	setInitialize();
+	setGetSystemLanguage(&lang);
+	setMakeLanguage(lang, &langcode);
+
+	switch (langcode) {
+		case SetLanguage_JA:   return "ja_JP";
+		case SetLanguage_ENUS: return "en_US";
+		case SetLanguage_FR:   return "fr_FR";
+		case SetLanguage_FRCA: return "fr_FR";
```
Comment (phcoder) [inline backends/platform/sdl/switch/switch.cpp:182]: Why fr_FR and not fr_CA?

File: backends/platform/sdl/switch/switch.cpp [hunk 2]
```diff
@@ -164,3 +166,29 @@ Common::HardwareInputSet *OSystem_Switch::getHardwareInputSet() {
...
+		case SetLanguage_DE:   return "de_DE";
+		case SetLanguage_IT:   return "it_IT";
+		case SetLanguage_ES:   return "es_ES";
+		case SetLanguage_ZHCN: return "zh_CN";
+		case SetLanguage_KO:   return "ko_KR";
+		case SetLanguage_NL:   return "nl_NL";
+		case SetLanguage_PT:   return "pt_PT";
+		case SetLanguage_RU:   return "ru_RU";
+		case SetLanguage_ZHTW: return "zh_HK";
```
Comment (phcoder) [inline backends/platform/sdl/switch/switch.cpp:191]: Why not zh_TW?

File: backends/platform/sdl/switch/switch.cpp (hunk 1 above)
Comment (usineur) [reply backends/platform/sdl/switch/switch.cpp:182]: fr_CA doesn't exist in LanguageDescription.

File: backends/platform/sdl/switch/switch.cpp (hunk 2 above)
Comment (usineur) [reply backends/platform/sdl/switch/switch.cpp:191]: yes, my mistake.

@rsn8887 can you fix that ?

**Pull request merged**